docs/managed_package.rst

```diff
@@ -1,27 +1,155 @@
-Release a managed package
+Release a Managed Package
 =========================
 
+This section outlines how to release a first generation (1GP) Salesforce Managed Package project. Salesforce.org's Release Engineering team practices `CumulusCI Flow<TODO link>`_, which incorporates all of these steps.
+
+
+
 Prerequisites
 -------------
 
-Deploy to a packaging org
+This section assumes:
+
+* CumulusCI is installed on your computer.
+* A Salesforce Managed Package project has been configured for use with CumulusCI.
+* A packaging org is connected to CumulusCI under the name of ``packaging``.
+
+
+
+Deploy to a Packaging Org
 -------------------------
 
-Upload and test a beta version
-------------------------------
+CumulusCI enables you to deploy to your ``packaging`` org.
+
+.. warning::
+
+    The ``ci_master`` flow runs the `uninstall_packaged_incremental<TODO>`_ task, which deploys destructive changes to the target org.
+
+.. code-block:: console
+
+    $ cci flow run ci_master --org packaging
+
+.. note::
+
+    This flow is `source format agnostic<TODO>`_.
+
+The ``ci_master`` flow executes these tasks in the target org.
+
+* Updates any project dependencies
+* Deploys any unpackaged metadata located in the ``pre`` directory
+* Deploys package metadata
+* Deploys destructive changes to remove metadata in the target org that is no longer in the local workspace
```

Comment:
Should we point out the individuals task names for each of these steps?

Reply:
I added a tip for listing all the steps.